lib/license_finder/cli/main.rb

```diff
@@ -175,6 +180,20 @@ def report_of(content)
       def save?
         !!config.save_file
       end
+
+      # Filter out proprietary dependencies (for license page only)
+      def filter_proprietary(dependencies)
+        return dependencies unless config.format == 'license-page'
+
+        dependencies.reject do |dep|
+          # Reject something we know about, so dependency should be either whitelisted or approved
+          if dep.whitelisted? || dep.approved?
+            name = dep.name
+            license = (dep.license_names_from_spec || [''])[0]
+            (license =~ /grand rounds/i) || (name == 'graphql-pro') || name.start_with?('grnds-')
```

Comment:
Good enough, I guess. I do think I found one other non-grnds thing we licensed, but I can't find it now.